src/BenchmarkDotNet/Running/BenchmarkConverter.cs

```diff
@@ -108,7 +115,6 @@ private static ImmutableConfig GetFullMethodConfig(MethodInfo method, ImmutableC
             Tuple<MethodInfo, TargetedAttribute>[] iterationCleanupMethods)
         {
             return targetMethods
-                .Where(m => m.HasAttribute<BenchmarkAttribute>())
```

Comment:
It's a regression.

Currently, this code silently filters wrong methods. After this code will throw friendly message (and exit) if any method is not benchmarkMethod.
```C#
BenchmarkRunner.Run(typeof(BenchmarkClass), new MethodInfo[] { BenchmarkClass.PrivateMethod, string.Concat });
```

Reply:
@YegorStepanov great catch, I've missed the fact that it's called by

```cs
public static BenchmarkRunInfo MethodsToBenchmarks(Type containingType, MethodInfo[] benchmarkMethods, IConfig config = null)
```

which contains user input

Reply:
I'm adding checks for BenchmarkRunner now: 

Should be BenchmarkRunner fast exit if any MethodInfo is wrong or it should print error to the console and keep running?

The same question for Type[]. (Note BenchmarkRunner.Run with `args != null` switches to BenchmarkSwitcher and exit with error)

```C#
BenchmarkRunner.Run(new Type[] {typeof(BenchmarkClass), typeof(int)}) 
```

Reply:
@YegorStepanov I think that for user provided `MethodInfo` it's OK to fail fast